tbx/blog/models.py

```diff
@@ -180,30 +180,35 @@ def services(self):
     def tags(self):
         return chain(self.services, self.sectors)
 
-    @property
+    @cached_property
     def related_blog_posts(self):
-        # format for template
-        return [
-            {
-                "title": blog_post.title,
```

Comment:
Thanks for updating these - andy had previously mentioned to me that he thought they were a bit overkill - but we didn't want to spend further time on that change at the time.